pkgs/fc/check-ceph/nautilus/tests/test_check_snapshot_restore.py

```diff
@@ -0,0 +1,202 @@
+import json
+import os
+import shutil
+from collections import namedtuple
+from copy import deepcopy
+from textwrap import dedent
+from unittest import mock
+
+import fc.check_ceph.check_snapshot_restore as snapcheck
+import pytest
+
+# fixtures and mocks are in conftest.py
+
+
+def test_fill_warnings(cluster_stats, snap_data, default_pool_roots):
+    thresholds = snapcheck.Thresholds(nearfull=0.85, full=0.95)
+    # retrieve and create mock data
+    largest_snaps = {
+        rootname: ("nopoolname", snap_data) for rootname in default_pool_roots
+    }
+    roots = list(default_pool_roots.keys())
+    # normal case: exit code 0
+    assert 0 == snapcheck.eval_fill_warnings(
```

Comment:
This would be a tiny bit more readable if this was semantic: assert STATE_OK = ...